Assets/Unity Simple Liquid/Scripts/LiquidContainer.cs

```diff
@@ -338,6 +349,35 @@ private void OnDrawGizmosSelected()
         }
         #endregion
 
+        #region ChangeColor
+        private List<LiquidContainer> listLC = new List<LiquidContainer>();
+        public List<LiquidContainer> GetColorList
+        {
+            get { return listLC; }
+        }
+
+        //playerMultiply for faster color change
+        [Range(0, 2)]
+        public float playerMultiply = 1;
```

Comment:
Can you please rename this field and add tooltip attribute? From current name it's hard to tell what it do.

Reply:
okay